pyaerocom/io/read_mscw_ctm.py

```diff
@@ -255,6 +259,16 @@ def read_var(self, var_name, ts_type=None, **kwargs):
         GriddedData
         """
 
+        if isinstance(ts_type, dict):
```

Comment:
In which case is ts_type a dictionary here? Can this happen in the higlevel tools? I.e. in `Colocator`?